BrainPortal/lib/cbrain_task_generators/schema_task_generator.rb

```diff
@@ -226,20 +228,23 @@ def register(task)
       # theres already one. Only applies to Bourreaux (as it would make no
       # sense on the portal).
       return if Rails.root.to_s =~ /BrainPortal\z/
-
       # Create a ToolConfig iff
-      #   (1) the Bourreau has a docker executable and
-      #   (2) the descriptor specifies a docker image
-      return if docker_image.nil? || !resource.docker_present
+      #   (1) the Bourreau has a container executable and
+      #   (2) the descriptor specifies a container image
+      return if container_engine.blank?  # Singularity or Docker
+      return if container_image.blank?   # image name or ID??
```

Comment:
In my slack comment I didn't mean to have the "image name or ID??" be a literal comment, I meant for you to replace it with the proper one...